packages/paste-libraries/use-clipboard/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@twilio-paste/use-clipboard-library",
```

Comment:
NIT: Could we consolidate the names for this package and the directory? Perhaps it makes sense to change the directory name and the name here to `use-clipboard`? FWICT I don't think we need to have the word "library" in the package name.

Reply:
Let's name this `"@twilio-paste/clipboard-copy-library"`

And yup, our libraries have a `-library` suffix to be more explicit and avoid any future name collisions (in case we release a component called `clipboard-copy`)